modules/test/playwright/pages/object-web/ModelBuilderPage.ts

```diff
@@ -41,10 +54,33 @@ export class ModelBuilderPage {
 		this.toggleSidebarsButton = page.getByLabel('Toggle Sidebars');
 	}
 
+	async deleteObjectRelationship(
+		objectRelationshipLabel: string,
+		objectRelationshipName: string
+	) {
+		this.clickObjectRelationshipEdge(objectRelationshipLabel);
+		await this.deleteObjectRelationshipButton.click();
+		await this.modalDeleteObjectRelationshipTextField.click();
+		await this.modalDeleteObjectRelationshipTextField.fill(
+			objectRelationshipName
+		);
+		await this.modalDeleteObjectRelationshipConfirmationButton.click();
+	}
+
+	async clickDeleteObjectRelationshipButton() {
+		this.deleteObjectRelationshipButton.click();
+	}
+
 	async clickFitViewButton() {
 		this.fitViewButton.click({force: true});
 	}
 
+	async clickObjectRelationshipEdge(objectRelationshipLabel: string) {
```

Comment:
Create a new method only if:
1 - It's called outside the POM
2 - It's a complex call and used more than once inside the POM.

The body of this function should be moved to `deleteObjectRelationship`.

Reply:
Hey @cgmonte, considering the reuse potential of the `clickObjectRelationshipEdge` function. Don't you think we could call the function outside the POM right before the `deleteObjectRelationship` function?

So:

```
	await modelBuilderPage.clickObjectRelationshipEdge(
		objectRelationshipLabel
	);

	await modelBuilderPage.deleteObjectRelationship(
		objectRelationshipLabel,
		objectRelationshipName
	);
```

Reply:
Yes, I agree. We should keep this method and call it from outside.